pkgs/development/tools/nimble/default.nix

```diff
@@ -1,37 +1,32 @@
-{ stdenv, fetchFromGitHub, nim }:
-
-let
-  srcs = {
-    nimble = fetchFromGitHub {
-      owner = "nim-lang";
-      repo = "nimble";
-      rev = "v0.7.2";
-      sha256 = "0j9b519cv91xwn6k0alynakh7grbq4m6yy5bdwdrqmc7lag35r0i";
-    };
-    nim = fetchFromGitHub {
-      owner = "nim-lang";
-      repo = "nim";
-      rev = "v0.13.0";
-      sha256 = "14grhkwdva4wmvihm1413ly86sf0qk96bd473pvsbgkp46cg8rii";
-    };
-  };
-in
+{ stdenv, fetchFromGitHub, nim, openssl, makeWrapper }:
+
 stdenv.mkDerivation rec {
   name = "nimble-${version}";
-  version = "0.7.2";
 
-  src = srcs.nimble;
+  version = "0.7.4";
+
+  src = fetchFromGitHub {
+    owner = "nim-lang";
+    repo = "nimble";
+    rev = "v${version}";
+    sha256 = "1l477f1zlqpc738jg47pz599cwjasgy9jqdsplj3ywd12xfqpc96";
+  };
 
-  buildInputs = [ nim ];
+  buildInputs = [ nim makeWrapper ];
 
-  postUnpack = ''
-    mkdir -p $sourceRoot/vendor
-    ln -s ${srcs.nim} $sourceRoot/vendor/nim
+  patchPhase = ''
+    substituteInPlace src/nimble.nim.cfg --replace "./vendor/nim" "${nim}/share"
   '';
-  buildPhase   = ''
+
+  buildPhase = ''
     nim c src/nimble
   '';
-  installPhase = "installBin src/nimble";
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp src/nimble $out/bin
+    wrapProgram $out/bin/nimble --prefix LD_LIBRARY_PATH : "${openssl.out}/lib"
```

Comment:
@ehmry I think the nimble package hash is fine. The travis output is strange, though. It looks like it is trying to build both derivations: nim and nimble, but it's using nimble's hash for nim:
```
unpacking source archive /run/user/1001/nix-build-nimble-v0.7.4-src.drv-0/v0.7.4.tar.gz

output path ‘/nix/store/v4bln6zap0f5n1yqdpdsdgjikbjd6z0y-nim-0.14.2.tar.xz’ has sha256 hash ‘15966078bmcijxi4gf7iw743q059n825dkjgkddl5f7rf3lvvgg7’ when ‘1yzjnz2jcis6l0wlxynqrjwzwrbk0gkhg15rivz8kmzf1a7b7g12’ was expected

cannot build derivation ‘/nix/store/waalv83bz6glk7cimnxvlqc93d5k9i78-nim-0.14.2.drv
```